docs/users/theory.md

```diff
@@ -1,132 +1,220 @@
+# Theory
 
-BespokeFit produces molecular mechanics parameters for user-specified chemicals. These parameters are "bespoke" to the chemical. Any process of simplifying chemical reality into a classical force field makes assumptions and trade-offs, and the resulting parameters are as much a result of the process as of the chemical. BespokeFit makes assumptions designed to allow accurate parameters for a SMIRNOFF force field to be generated automatically and performantly. The process it uses and the theory behind it are documented here.
+BespokeFit aims to produce highly accurate, "bespoke" classical molecular force field parameters for user-specified 
+molecules, or even series of molecules that share common features such is common in lead series. In the page we will 
+describe the theory behind the workflow used to produce such parameters, as well as any assumptions and trade-offs made.
 
-A BespokeFit run goes through five steps:
+The process by which bespoke parameters are generated generally follows five main stages:
 
-1. **Parameter selection** - The target parameters are selected and reference values are taken from the initial force field
-2. **Fragmentation** - Chemicals are broken up around the target parameters for faster quantum chemical calculations
-3. **SMIRKS generation** - SMIRKS patterns are constructed to identify the chemistry associated with the target parameters
-4. **QC generation** - Reference data are generated via quantum chemical calculations 
-5. **Parameter optimization** - The parameters are fitted to the reference QC data
+1. **Parameter selection** - important features of the molecule(s) that require bespoke parameters, such as rotatable 
+   bonds, are identified
+2. **Fragmentation** - molecules are split into smaller fragments based on the identified features for faster quantum 
+   chemical calculations
+3. **QC generation** - any required quantum chemical reference data is generated using the smaller chemical fragments 
+4. **Parameter generation** - bespoke SMIRKS patterns that match the identified chemical features are constructed and 
+   initial values sourced from a general force field
+5. **Parameter optimization** - the bespoke parameters are trained to the reference QC data
 
 ## Parameter selection
 
-A BespokeFit run begins by selecting target parameters from the target molecule. By default, BespokeFit only optimizes torsion parameters, and selects every rotatable bond whose torsion angle can be constructed without considering terminal hydrogen atoms. Other parameters can be chosen by setting the [`target_templates`] and [`target_torsion_smirks`] fields of the workflow factory.
+The first stage in the bespoke fitting workflow is identifying those features of a molecule that require parameters with 
+high specificity in order to achieve the best accuracy. 
+
+At present, this involves identifying all non-terminal rotatable bonds in a molecule. Accurately reproducing the torsion profile 
```

Comment:
I really like the addition of the "no triple bonds to neighbors and no ring bonds" below though, that should definitely be kept.

Reply:
The previous wording wasn't quite correct either unfortunately - I've added a sentence to clarify. In future it may be worth compiling a glossary of common terms such as this that we can just link to _a la_ [OpenEye](https://docs.eyesopen.com/toolkits/python/oechemtk/glossary.html#term-rotatable-bond)